src/main/java/xbot/common/controls/actuators/XCANMotorController.java

```diff
@@ -293,4 +484,17 @@ protected boolean isValidPowerRequest(double power) {
         }
         return true;
     }
+
+    protected <N extends Unit, D extends Unit> Measure<? extends PerUnit<D, N>> invertRatio(Measure<? extends PerUnit<N, D>> ratio) {
+        return ratio.unit().reciprocal().of(1 / ratio.magnitude());
```

Comment:
We either need divide-by-zero protection here, or sanitization on the setting side to ensure that `magnitude` would never be zero.